components/Cards/Card.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import AllOutIcon from "@material-ui/icons/AllOut";
+import DoneIcon from "@material-ui/icons/Done";
+import EcoIcon from "@material-ui/icons/Eco";
+import LockIcon from "@material-ui/icons/Lock";
+
+const Style = "text-white text-xs";
+
+const arrayIcon = [
+  <AllOutIcon fontSize="small" className={Style} />,
+  <DoneIcon fontSize="small" className={Style} />,
+  <EcoIcon fontSize="small" className={Style} />,
+  <LockIcon fontSize="small" className={Style} />,
+];
+const Color = [
+  "from-indigo-500 to-blue-500",
+  "from-blue-400 to-blue-300",
+  "from-green-500 to-green-400",
+  "from-yellow-600 to-yellow-500",
+];
+
+const TemplateCard = (props) => {
+  var balance = props.balance;
+  balance = balance * 9.5;
+
+  return (
+    <div
+      className={`transform hover:scale-110 cursor-pointer transition delay-100 w-3/12  p-2 py-4 shadow-xl  border rounded-xl bg-gradient-to-r ${
```

Comment:
look into classnames npm package: https://www.npmjs.com/package/classnames This will result in cleaner code